### ISSUE TITLE
"Log" link is not available when a job errors

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-ui#43]
When a job errors is when the user needs the log the most. Usually there is even a message saying "Please see detailed log for more info". I'm not sure whether this is a UI or server issue.

**Comment by marisademeglio** [comment]
Yeah I noticed this too. Even when a job successfully finishes, the web service does not return a log link, so I had to make a workaround for this, but it probably isn't getting invoked when the job does not complete. 

That is to say: I'm pretty sure it's a server issue.

**Comment by bertfrees** [comment]
> Even when a job successfully finishes, the web service does not return a log link

You mean does not always return a log link? Or never? It works fine in my tests. Could it be a problem in the backend of the UI?

**Comment by marisademeglio** [comment]
Hmm I don't remember the exact details of the issue I found but there was something weird with web service and log file links. 

But anyway, to get to the root of what you're seeing -- So the web service returns a log link for a job with ERROR status but the log link is not showing up in the UI?

**Comment by bertfrees** [comment]
> So the web service returns a log link for a job with ERROR status

No, it doesn't :-)
